common/models.py

```diff
@@ -0,0 +1,107 @@
+import logging
+
+from django.db import models
+from django.utils import timezone
+from django.conf import settings
+
+LOGGER = logging.getLogger(__file__)
+
+CONTACT_TYPES = (
+    ('EMAIL', 'email'),
+    ('LANDLINE', 'landline'),
+    ('MOBILE', 'Mobile Phone number'),
+
+)
+
+
+class AbstractBase(models.Model):
+    """
+    Provides auditing attributes to a model.
+    It will provide audit fields that will keep track of when a model
+    is created or updated and by who.
+    """
+
```

Comment:
It is good that every model descends from `AbstractBase` ( enforce that rule firmly ).

Remember why we moved to Django 1.8 even before it was released? Because we wanted **UUID primary keys** - see https://docs.djangoproject.com/en/1.8/ref/models/fields/#django.db.models.UUIDField 

( add an explicit `id` field ).

These fields will make federation much easier when the time for that comes.